src/win/fs-event.c

```diff
@@ -80,7 +80,7 @@ static void uv__relative_path(const WCHAR* filename,
   (*relpath)[relpathlen] = L'\0';
 }
 
-static int uv__split_path(const WCHAR* filename, WCHAR** dir,
+int uv__fs_event_split_path(const WCHAR* filename, WCHAR** dir,
```

Comment:
This function is not really related to fs events, I think it'd be better named `uv__fs_split_path` and moved to fs.c.